packages/psss/src/views/Tabs/AlertsTabView.js

```diff
@@ -19,41 +31,62 @@ const DateSubtitle = styled(Typography)`
 
 export const AlertsTabView = React.memo(() => {
   const [isPanelOpen, setIsPanelOpen] = useState(false);
-  const { countryCode } = useParams();
+  const [isPickerOpen, setIsPickerOpen] = useState(false);
+  const [rowData, setRowData] = useState({});
+  const [period, setPeriod] = useState(getCurrentPeriod());
+  const { organisationUnit: countryCode, syndromeName } = rowData;
 
-  const handlePanelOpen = useCallback(() => {
-    setIsPanelOpen(true);
-  }, [setIsPanelOpen]);
+  const handleTableRowClick = useCallback(
+    data => {
+      setIsPanelOpen(true);
+      setRowData(data);
+    },
+    [setIsPanelOpen, setRowData],
+  );
 
   const handlePanelClose = useCallback(() => {
     setIsPanelOpen(false);
   }, [setIsPanelOpen]);
 
+  const DatePicker = (
+    <>
+      <CalendarButton onClick={() => setIsPickerOpen(true)}>
+        <CalendarTodayIcon />
+      </CalendarButton>
+      <WeekPicker
+        label="Date"
+        onChange={newDate => setPeriod(getPeriodByDate(newDate))}
+        value={getDateByPeriod(period)}
+        isOpen={isPickerOpen}
+        onClose={() => setIsPickerOpen(false)}
+      />
+    </>
+  );
+
   return (
     <div style={{ position: 'relative' }}>
```

Comment:
I think this div could be removed. I'd say it was just to hold the placeholder coming soon banner

Reply:
Great point!